File: Java/Practice/Addition.java
import java.util.Scanner;
class Addition
{
    public static void main(String[] args)
    {
        int a, b, c;
        Scanner scan = new Scanner(System.in);
        System.out.print("Enter a number: ");
        a = scan.nextInt();
        System.out.print("Enter another number: ");
        b = scan.nextInt();
        c = a + b;
        System.out.println("The sum is " + c);
    }
} 

File: C Programs/practice/Addition.c
#include<stdio.h>
int main()
{
    int a, b, c;
    printf("Enter a number");
    scanf("%d", &a);
    printf("Enter anoter number");
    scanf("%d", &b);
    c = a + b;
    printf("The sum is %d\n" , c);
    return 0;
}